Main.java

```diff
@@ -0,0 +1,66 @@
+package com.company;
+
+import java.io.*;
+import java.util.StringTokenizer;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        if (args.length != 2) {
+            System.out.println("File path and search word cannot be empty");
+            return;
+        }
+        String textFilePath = args[0];
+        String searchKeyword = args[1];
+        File file = new File(textFilePath);
+        System.out.println("Processing...");
+        validateFile(file);
+        if (validateFile(file)) {
+            wordCount(file, searchKeyword);
+        }
+    }
+
+    private static boolean validateFile(File file) {
+
+        if (!file.exists()) {
+            System.out.println("File does not exist");
+
+        } else if (!(file.getName().endsWith(".txt") || file.getName().endsWith(".json"))) {
+            System.out.println("File format not supported");
+            System.exit(0);

```

Comment:
Do not use System.exit(), just return a boolean value to indicate whether the file is valid or not, and in the main method check for the return value and proceed accordingly.

Reply:
Sure. I will do that.

Reply:
Still using exit() method here. Just return false here

Reply:
I have removed the explicit exit() statement and used return instead.